src/main/java/org/sagebionetworks/web/client/widget/entity/editor/VimeoConfigEditor.java

```diff
@@ -21,7 +21,6 @@
 	@Inject
 	public VimeoConfigEditor(IFrameConfigView view) {
 		this.view = view;
-		view.setPresenter(this);
```

Comment:
Why remove this setPresenter?  It seems dangerous for the injected view not to have a valid reference to this presenter.